sdk/modules/ui/cpp/src/base_glfw_window.cpp

```diff
@@ -46,11 +46,11 @@ struct GLFWContext {
             throw std::runtime_error("Impossible to create a glfw window");
 
         glfwMakeContextCurrent(window);
-
+#if defined(__linux__)
         if (glewInit() != GLEW_OK) {
```

Comment:
Is there a reason this function is not available for MacOS?
Anyway, we still want to call this on windows.

The best way to do what you want is to mimic what we did for GLES:
https://github.com/prophesee-ai/openeb/blob/main/sdk/modules/ui/cpp/include/metavision/sdk/ui/utils/opengl_api.h#L20
And provide a dummy implementation of `glewInit()` only for MacOS. It will avoid too much `#if` in actual implementation.

Reply:
Thanks! This makes a lot more sense now. In the midst of tackling the initial barrage of compiler errors, I missed this implementation hidden under GLES.
I also didn't realize that once I added `#include <OpenGL/gl3.h>` I didn't need the APPLE specific versions of the OpenGL calls anymore!

Easy changes to make and the MacOS build is still working. Same for the comments below.